docs/performance/inserts/sequences.rst

```diff
@@ -0,0 +1,182 @@
+.. _autogenerated_sequences_performance:
+
+###########################################################
+ Autogenerated sequences and PRIMARY KEY values in CrateDB
+###########################################################
+
+If you are coming to CrateDB from systems like PostgreSQL or MySQL one thing you
```

Comment:
I have tried to rephrase the intro/motivation a bit:

As you begin working with CrateDB, you might be puzzled why CrateDB does not have a built-in, auto-incrementing "serial" data type as PostgreSQL or MySQL.

As a distributed database , designed to scale horizontally, CrateDB needs as many operations as possible to complete independently on each node without any coordination between nodes. 

Maintaining a global auto-increment value will require that a node would need to check with all other nodes before allocating a new value - and preventing all other nodes to allocate new values. This bottleneck will be hindering our ability to achieve extremely_fast_ingestion_speeds.